Src/AutoNSubstitute/AutoNSubstitute.csproj

```diff
@@ -64,6 +64,7 @@
   <ItemGroup>
     <Compile Include="AutoConfiguredNSubstituteCustomization.cs" />
     <Compile Include="AutoNSubstituteCustomization.cs" />
+    <Compile Include="AutoSubstituteCustomization.cs" />

```

Comment:
Could we think of another name for this class?

The name makes sense, but it only differs by a single character from `AutoNSubstituteCustomization`, so I'm a bit concerned that newcomers would mistype the name, or not understand the difference.

That would give a bad first impression of the overall library, and lead to many questions here or on Stack Overflow.